src/lws/lws-wallet-connection-error.tsx

```diff
@@ -0,0 +1,74 @@
+import * as React from 'react';
+import injectSheet, { StyleSheet } from 'react-jss';
+import { LWSButton } from './lws-button';
+import CommonStyle from '../common/common-style';
+import { WalletIcon } from '../icons/wallet';
+
+
+const styles: StyleSheet = {
+  buttonPrimary: CommonStyle.buttonPrimary,
+
+  buttonSecondary: CommonStyle.buttonSecondary,
+
+  form: CommonStyle.form,
+
+  formSubmitRow: {
+    marginTop: '15px',
+    '& button': {
+      marginTop: '30px'
+    }
+  },
+
+  supportText: {
+    textAlign: 'center',
+    lineHeight: '22px',
+    fontFamily: 'ProximaNovaSemibold',
+    color: '#C5DCE9',
+    padding: '0 0 30px'
+  },
+
+  areaTitle: {
+    textAlign: 'center',
+    '& h2': {
+      textTransform: 'uppercase',
+      fontWeight: 'normal',
+      fontSize: '16px',
+      padding: '25px 0',
+      margin: 0,
+      color: '#D97300'
+    },
+    '& h3': {
+      fontWeight: 'normal',
+      fontSize: '21px',
+      padding: '0 0 45px',
+      margin: 0,
+      color: '#93B0C1'
+    }
+  },
+};
+
+export type LWSWalletConnectionErrorProps = {
+  downloadWalletAction?: ((event: React.MouseEvent<HTMLElement>) => void),
+  retryAction?: ((event: React.MouseEvent<HTMLElement>) => void),
+};
+
+export const LWSWalletConnectionError = injectSheet(styles)<LWSWalletConnectionErrorProps>(({classes, children, downloadWalletAction, retryAction}) => (
+  <div>
+      <div className={classes.areaTitle}>
+          <WalletIcon/>
+          <h2>Action Required</h2>
+          <h3>Install & Open The SelfKey Identity Wallet</h3>
+      </div>
+      <div className={classes.form}>
+        <p className={classes.supportText}>
+            The SelfKey Identity Wallet is required to securely authenticate with this website. Please download and open the SelfKey Identity Wallet to proceed. 
```

Comment:
at what point should we move static text like this to a string file for future translation support?

Reply:
@altninja we talked about localization before and it can be added pretty easily in react with https://www.i18next.com/ as suggested by @dfcowell. cheers